Wikipedia/Code/ArticleViewController.swift

```diff
@@ -1055,7 +1063,55 @@ private extension ArticleViewController {
         setupWebView()
         setupMessagingController()
     }
-    
+
+    private var overflowMenu: UIMenu {
+        let learnMore = UIAction(title: CommonStrings.learnMoreTitle(), image: UIImage(systemName: "info.circle"), handler: { [weak self] _ in
+            self?.goToFAQ()
+        })
+        
+        let tutorial = UIAction(title: CommonStrings.tutorialTitle, image: UIImage(systemName: "lightbulb.min"), handler: { [weak self] _ in
+            self?.showTutorial()
+        })
+
+        let reportIssues = UIAction(title: CommonStrings.problemWithFeatureTitle, image: UIImage(systemName: "flag"), handler: { [weak self] _ in
+            self?.reportIssue()
+        })
+
+        let menuItems: [UIMenuElement] = [learnMore, tutorial, reportIssues]
+
+        return UIMenu(title: String(), children: menuItems)
+    }
+
+    private func goToFAQ() {
+        if let altTextExperimentViewModel {
+            navigate(to: altTextExperimentViewModel.learnMoreURL, useSafari: false)
+        }
+    }
+
+    private func showTutorial() {
+        // presentTooltipsIfNecessary(onBottomSheetViewController: imageRecommendationBottomSheetController, force: true)
+    }
+
+    private func reportIssue() {
+        func imageRecommendationsUserDidTapReportIssue() {
+            let emailAddress = "ios-support@wikimedia.org"
+            let emailSubject = WMFLocalizedString("image-recommendations-email-title", value: "Issue Report - Alt Text Feature", comment: "Title text for Image recommendations pre-filled issue report email")
+            let emailBodyLine1 = WMFLocalizedString("image-recommendations-email-first-line", value: "I've encountered a problem with the Alt Text feature:", comment: "Text for Image recommendations pre-filled issue report email")
+            let emailBodyLine2 = WMFLocalizedString("image-recommendations-email-second-line", value: "- [Describe specific problem]", comment: "Text for Image recommendations pre-filled issue report email. This text is intended to be replaced by the user with a description of the problem they are encountering")
+            let emailBodyLine3 = WMFLocalizedString("image-recommendations-email-third-line", value: "The behavior I would like to see is:", comment: "Text for Image recommendations pre-filled issue report email")
+            let emailBodyLine4 = WMFLocalizedString("image-recommendations-email-fourth-line", value: "- [Describe proposed solution]", comment: "Text for Image recommendations pre-filled issue report email. This text is intended to be replaced by the user with a description of a user suggested solution")
+            let emailBodyLine5 = WMFLocalizedString("image-recommendations-email-fifth-line", value: "[Screenshots or Links]", comment: "Text for Image recommendations pre-filled issue report email. This text is intended to be replaced by the user with a screenshot or link.")
```

Comment:
For the strings here from line 1097 forward:

If the copy is the same, you can move it into CommonStrings, as we talked about during eng sync. You can keep the keys the same so they don't need to be translated again (changing keys will kick off that whole process as it was a new string).